cpp/src/arrow/compute/CMakeLists.txt

```diff
@@ -26,8 +26,22 @@ arrow_add_pkg_config("arrow-compute")
 # Unit tests
 #
 
+# As of 2023-02-28, the following kernels are always present:
```

Comment:
```suggestion
# The following kernels are always present:
```
I'm not sure the date is useful (and it will be hard to keep correct).  Users can always use git blame if they need it.